FrameworkSystem/Service/PlottingHandler.py

```diff
@@ -29,7 +29,7 @@ def initializePlottingHandler(serviceInfo):
     pass
   try:
     testFile = "%s/plot__.test" % dataPath
-    fd = file(testFile, "w")
+    fd = open(testFile, "w")
```

Comment:
```suggestion
    with open(testFile, "w"):
      pass
```
and delete the next line